infra/tf-next/firewall.tf

```diff
@@ -0,0 +1,81 @@
+resource "digitalocean_firewall" "ursa_peering" {
+  name = "ursa-peering"
+
+  tags = ["ursa-main", "ursa-bootstrap"]
+
+  # Allow SSH
+  inbound_rule {
+    protocol         = "tcp"
+    port_range       = "22"
+    source_addresses = ["0.0.0.0/0"]
+  }
+
+  # Allow Tcp p2p port for ursa node
+  inbound_rule {
+    protocol         = "tcp"
+    port_range       = "6009"
+    source_addresses = ["0.0.0.0/0", "::/0"]
+  }
+
+  # Allow Quic p2p port for ursa node
+  inbound_rule {
+    protocol         = "udp"
+    port_range       = "6009"
+    source_addresses = ["0.0.0.0/0", "::/0"]
+  }
+
+  # Allow HTTP/HTTPS ingress from load balancer
+  # todo(botch): custom load balancer port
+  # inbound_rule {
+  #   protocol         = "tcp"
+  #   port_range       = "443"
+  #   source_addresses = [digitalocean_loadbalancer.main.id]
+  # }
+
+  # Allow HTTP/HTTPS ingress
+  inbound_rule {
+    protocol         = "tcp"
+    port_range       = "80"
+    source_addresses = ["0.0.0.0/0", "::/0"]
+  }
+
+  inbound_rule {
+    protocol         = "tcp"
+    port_range       = "443"
+    source_addresses = ["0.0.0.0/0", "::/0"]
+  }
+
+  # Allow Tcp p2p port for ursa node
+  outbound_rule {
+    protocol              = "tcp"
+    port_range            = "6009"
+    destination_addresses = ["0.0.0.0/0", "::/0"]
+  }
+
+  # Allow Quic p2p port for ursa node
+  outbound_rule {
+    protocol              = "udp"
+    port_range            = "6009"
+    destination_addresses = ["0.0.0.0/0", "::/0"]
+  }
```

Comment:
```suggestion
```

No need for this since allowing all ports just below

Reply:
You are right but for devnet I have it explicit because after devnet I'll restrict to just the 6009 port. As now it's not clear what port the gateway needs to call the indexer. What the metrics needs to call out so, etc.

Reply:
I think though that the node will open random outbound ports right? not sure if we can limit to a range or something for that. But makes sense :100: